nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -567,40 +565,29 @@ public void testGCEnabled() {
   }
 
   private String getTableBasePath(String tableName) {
-    String databasePath = temp.toString() + "/" + DB_NAME;
-    return Paths.get(databasePath, tableName).toAbsolutePath().toString();
-  }
-
-  protected Path getTableLocationPath(String tableName) {
-    return new Path("file", null, Paths.get(getTableBasePath(tableName)).toString());
-  }
-
-  protected String getTableLocation(String tableName) {
-    return getTableLocationPath(tableName).toString();
-  }
-
-  private String metadataLocation(String tableName) {
-    return Paths.get(getTableBasePath(tableName), "metadata").toString();
+    return temp.toURI() + DB_NAME + "/" + tableName;
```

Comment:
Does this automatically call `toString` on the URI from `temp`?

Reply:
Yes, apparently this is what it compiles to:
`javap -cp nessie/build/classes/java/test/ -p -c -s org.apache.iceberg.nessie.TestNessieTable`

```
  private java.lang.String getTableBasePath(java.lang.String);
    descriptor: (Ljava/lang/String;)Ljava/lang/String;
    Code:
       0: new           #88                 // class java/lang/StringBuilder
       3: dup
       4: invokespecial #89                 // Method java/lang/StringBuilder."<init>":()V
       7: aload_0
       8: getfield      #90                 // Field temp:Ljava/io/File;
      11: invokevirtual #91                 // Method java/io/File.toURI:()Ljava/net/URI;
      14: invokevirtual #92                 // Method java/lang/StringBuilder.append:(Ljava/lang/Object;)Ljava/lang/StringBuilder;
      17: ldc           #93                 // String db
      19: invokevirtual #94                 // Method java/lang/StringBuilder.append:(Ljava/lang/String;)Ljava/lang/StringBuilder;
      22: ldc           #95                 // String /
      24: invokevirtual #94                 // Method java/lang/StringBuilder.append:(Ljava/lang/String;)Ljava/lang/StringBuilder;
      27: aload_1
      28: invokevirtual #94                 // Method java/lang/StringBuilder.append:(Ljava/lang/String;)Ljava/lang/StringBuilder;
      31: invokevirtual #96                 // Method java/lang/StringBuilder.toString:()Ljava/lang/String;
      34: areturn
```

but it's different with java 11, something like:
```
24: invokedynamic #9,  0              // InvokeDynamic #0:makeConcatWithConstants:(Ljava/net/URI;)Ljava/lang/String;
```